js-pkg/driftdb/src/types.ts

```diff
@@ -37,7 +37,7 @@ export type MessageToDb = {
 } | {
     type: 'get'
     key: Key
-    seq: SequenceNumber
+    seq?: SequenceNumber
```

Comment:
Without a sequence number this returns the whole list of messages?

Reply:
Yes. When the message is parsed on the server, if `seq` is missing it (now) uses the default value of 0. The sequence number in a `get` message is a lower bound, so all messages are returned.